testing/features/automated_release_process.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Automated Release Process
+
+This feature is to automate the design system release process, removing repeatable manual developer actions. 
+
+As a design system developer
+I need to be able to release updates to the design system using an automated process 
+So that I can save time and improve the consistency of release artefacts and processes
+
+As a design system developer 
+I need to be able to specify that a design system release is "pre-release" or "stable release"
+So that users understand that the design system is ready for either the development or production environments
```

Comment:
Is the developer the user here, or is the person consuming the design system the user? E.g.

```
As a developer using the design system components
I need to know whether a given release is "pre-release" or "stable"
So that I know whether to use it in production-ready code
```

Reply:
Yes - good question and I think your version better captures the core need, will update.

Reply:
re you're first point I'll also update to be "As a maintainer of the design system components"

On the repo renaming, I think the more specific we can be the better, but will leave to others to change the name in case I break it!

Reply:
Oh yeah - I don't think we should actually rename the repo... just what we refer to it as in these requirements. But that's one for another day!